src/App.jsx

```diff
@@ -14,70 +15,144 @@ export const goods = [
+export const App = () => {
+  const [value, setValue] = useState(['Jam']);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  // console.log(value);
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  const addValue = (good) => {
+    if (value.length === 0) {
+      setValue(good);
+    }
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+  const removeValue = () => {
+    setValue('');
+  };
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+  return (
+    <main className="section container">
+      { value === '' && (
+      <h1 className="title is-flex is-align-items-center">
+        No goods selected
+      </h1>
+      )}
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+      {value && (
+        <h1 className="title is-flex is-align-items-center">
+          {`${value} is selected`}
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => setValue('')}
+          />
+        </h1>
+      )}
 
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+      <table className="table">
+        <tbody>
+          <tr
+            data-cy="Good"
+            className={`${value.includes('Dumplings') ? 'has-background-success-light' : ''}`}
+          >
+            <td>
+              {value.includes('Dumplings')
+                ? (
+                  <button
+                    data-cy="RemoveButton"
+                    type="button"
+                    className="button"
+                    onClick={() => removeValue('Dumplings')}
+                  >
+                    -
+                  </button>
+                )
+                : (
+                  <button
+                    data-cy="AddButton"
+                    type="button"
+                    className="button is-info"
+                    onClick={() => addValue('Dumplings')}
+                  >
+                    +
+                  </button>
+                )}
```

Comment:
look like strange logic, why Dumplings? why do you need this

Reply:
still this logic working with `Dumplings `. You should work with all items you got